src/PVRIptvData.cpp

```diff
@@ -469,12 +470,23 @@ bool PVRIptvData::LoadPlayList(void)
         tmpChannel.properties.insert({prop, propValue});
       }
     }
+    else if (StringUtils::Left(strLine, (int)strlen(PLAYLIST_TYPE_MARKER)) == PLAYLIST_TYPE_MARKER)
```

Comment:
Please use a c++ style cast here. I see you copied this from some lines above, but please don't copy blindly.

Reply:
I had a look at the signature of StringUtils::Left and it actually takes a parameter of type size_t which is exactly what strlen returns.
So the cast is useless at all - will remove it for the other occurrences as well.